examples/closed-issue-counter/closed_issue_counter.bal

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/lang.'int as integer;
+import ballerina/task;
+import ballerina/time;
+import ballerina/jballerina.java;
+
+listener http:Listener issueCounter = new (9092);
+
+service /scheduler on issueCounter {
+
+    resource function post recurJob/[decimal interval](@http:Payload json config) returns error|int {
+        string startTime = (check config.startTime).toString();
+        string repeatingCount = (check config.repeatingCount).toString();
```

Comment:
what if the payload doesn't contain those two fields

Reply:
Updated the coding by removing if clauses